corehq/apps/smsbillables/migrations/0023_auto_20210107_1609.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2021-01-07 16:09
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsbillables', '0022_pinpoint_gateway_fee_amount_null'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='smsbillable',
+            name='direct_gateway_fee',
+            field=models.DecimalField(decimal_places=5, max_digits=10, null=True),
```

Comment:
I hope this is a safe/quick migration to apply to prod, but I am not sure. There is a chance it will need to rewrite the entire table, which might take a long time and cause other issues. Have you looked into that?